packages/adapter-nextjs/src/index.ts

```diff
@@ -2,3 +2,4 @@
 // SPDX-License-Identifier: Apache-2.0
 
 export { createServerRunner } from './createServerRunner';
+export { NextServer } from "./types";
```

Comment:
nit: using single quote and add a new line to the end of file
```suggestion
export { NextServer } from './types';
```